templates/networkpolicies.yaml

```diff
@@ -90,6 +94,7 @@ spec:
       ports:
         - protocol: TCP
           port: 5432
-  # Deny all egress traffic
-  egress: []
+  # Explicitly allow all egress traffic
+  egress:
```

Comment:
I think we should only allow egress to Istio here, since otherwise timescale really has no need to initiate connections? Admins that want to allow all can disable the policies entirely

Reply:
We can add explicit rules for individual service meshes, but we won't know in advance which ones folks will be using.

Reply:
Yeah, but the workaround is simple (disable network policies, with the corresponding Helm value) and we can always extend the rules later if needed. I'd prefer to keep things locked down as much as possible